.circleci/workflows.yml

```diff
@@ -198,6 +201,63 @@ jobs:
               -f services/delegation/charts/community-delegation/values.yaml \
               -f services/delegation/charts/${HELM_CONFIG} \
               --set image.tag=${VERSION}
+  build-claim-backend:
+    <<: *config
+    steps:
+      - *step_install_deps
+      - checkout
+      - *step_setup_remote_docker
+      - *step_prepare_environment
+      - run:
+          name: Build image
+          shell: /bin/bash -eo pipefail
+          command: |
+            docker build -t taraxa/claim-backend:${VERSION} -f services/claim-backend/Dockerfile .
+      - run:
+          name: Tag and push images
+          shell: /bin/bash -eo pipefail
+          command: |
+            if [ "${ENV}" == "prod" ]; then
+              echo ${DOCKERHUB_PASS} | docker login -u taraxa --password-stdin
+
+              docker tag taraxa/claim-backend:${VERSION} taraxa/claim-backend:latest
+              docker push taraxa/claim-backend:${VERSION}
+              docker push taraxa/claim-backend:latest
+            else
+              echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
+
+              docker tag taraxa/claim-backend:${VERSION} ${GCP_REGISTRY}/taraxa/claim-backend-develop:${VERSION}
```

Comment:
You changed the image name. It used to be `taraxa-claim-api-develop`. I'm fine with that. I'm all for having the same name everywhere. I just wanted to add a note here to delete the old registry.